### PR TITLE
clean up remaining TODOs in volumeFilter_test.go

### PR DESCRIPTION
**Pull request by nikhilsaraf**
as title suggests

### CODE REVIEW

File: plugins/volumeFilter_test.go [hunk 1]
```diff
@@ -3,16 +3,17 @@ package plugins
 import (
 	"database/sql"
 	"fmt"
+	"strings"
```
Comment (nikhilsaraf) [inline plugins/volumeFilter_test.go:6]: only "strings" added. rest is adjustments made by IDE

File: plugins/volumeFilter_test.go [hunk 2]
```diff
@@ -38,7 +39,6 @@ func TestMakeFilterVolume(t *testing.T) {
 	testAssetDisplayFn := model.MakeSdexMappedAssetDisplayFn(map[model.Asset]hProtocol.Asset{model.Asset("XLM"): utils.NativeAsset})
 	configValue := ""
 	tradingPair := &model.TradingPair{Base: "XLM", Quote: "XLM"}
-	modes := []volumeFilterMode{volumeFilterModeExact, volumeFilterModeIgnore}
```
Comment (nikhilsaraf) [inline plugins/volumeFilter_test.go:41]: inlined into test below

File: plugins/volumeFilter_test.go [hunk 3]
```diff
@@ -47,7 +47,6 @@ func TestMakeFilterVolume(t *testing.T) {
 		wantMarketIDs []string
 		wantFilter    *volumeFilter
 	}{
-		// TODO DS Confirm the empty config fails once validation is added to the constructor
```
Comment (nikhilsaraf) [inline plugins/volumeFilter_test.go:50]: added separate test for this below

File: plugins/volumeFilter_test.go [hunk 4]
```diff
@@ -99,12 +98,12 @@ func TestMakeFilterVolume(t *testing.T) {
...
 
+	caseNo := 1
 	for _, k := range testCases {
 		// this lets us test both types of modes when varying the market and account ids
-		for _, m := range modes {
+		for _, m := range []volumeFilterMode{volumeFilterModeExact, volumeFilterModeIgnore} {
 			// this lets us test both buy and sell
-			// TODO DS Add buy action
-			for _, action := range []queries.DailyVolumeAction{queries.DailyVolumeActionSell} {
+			for _, action := range []queries.DailyVolumeAction{queries.DailyVolumeActionSell, queries.DailyVolumeActionBuy} {
```
Comment (nikhilsaraf) [inline plugins/volumeFilter_test.go:106]: added buy action, completing TODO

File: plugins/volumeFilter_test.go [hunk 5]
```diff
@@ -129,9 +128,10 @@ func TestMakeFilterVolume(t *testing.T) {
 						configType = "base"
 					}
 
-					// TODO DS Vary filter action between buy and sell, once buy logic is implemented.
 					wantFilter := makeWantVolumeFilter(config, k.wantMarketIDs, k.accountIDs, action)
-					t.Run(fmt.Sprintf("%s/%s/%s", k.name, configType, m), func(t *testing.T) {
+					testCaseInstanceName := fmt.Sprintf("%d. %s,%s,%s,%s", caseNo, k.name, configType, m, action.String())
```
Comment (nikhilsaraf) [inline plugins/volumeFilter_test.go:132]: restructured naming for convenience, adding action.String() at the end

File: plugins/volumeFilter_test.go [hunk 6]
```diff
@@ -155,6 +155,28 @@ func TestMakeFilterVolume(t *testing.T) {
...
+		"someConfigValue",
+		"someExchangeName",
+		tradingPair,
+		testAssetDisplayFn,
+		utils.NativeAsset,
+		utils.NativeAsset,
+		&sql.DB{},
+		configUnderTest,
+	)
+	if !assert.Error(t, e) {
```
Comment (nikhilsaraf) [inline plugins/volumeFilter_test.go:173]: new test with an empty config should return an error as name of test suggests

**Review by debnil — APPROVED** [review]
LGTM! Thanks for cleaning these up :)

**Pull request merged**